app/models/data_resources/resource_modules/category.rb

```diff
@@ -15,45 +15,27 @@ class Category < ApplicationRecord
 
   after_destroy :cleanup_data_resource_settings
 
-  def generic_items_count
-    return 0 if generic_items.blank?
-
-    generic_items.visible.count
-  end
-
-  def points_of_interest_count
-    return 0 if points_of_interest.blank?
-
-    points_of_interest.visible.count
-  end
-
-  def tours_count
-    return 0 if tours.blank?
-
-    tours.visible.count
-  end
-
-  def news_items_count
-    return 0 if news_items.blank?
-
-    news_items.visible.count
-  end
```

Comment:
`news_items_count` is entirely removed but not listed in the symbols for generating methods.

Reply:
i even had not news_items in my graphql test.
so i will readd the method as before!

Reply:
fixed by readding lost method